app/controllers/api/v2/users_controller.rb

```diff
@@ -33,10 +33,16 @@ def index
 
       api :GET, "/users/:id/", N_("Show a user")
       param :id, String, :required => true
+      api :GET, "/current_user", N_("Show the currently logged-in user")
```

Comment:
This should be above the method, which serves the request.

Reply:
Got it, thanks @ezr-ondrej